_pages/en_US/basic-information.txt

```diff
@@ -0,0 +1,76 @@
+---
+title: "Basic Information"
+---
+
+{% include toc title="Table of Contents" %}
+
+If this is your first time modding a 3DS (or you're unfamiliar with technology), keep reading! Otherwise, if you have prior experience and know what you're doing, skip to [Get Started](get-started).
+{: .notice--info}
+
+This page covers some of the base computer terminology and 3DS-specific quirks that you should know before following this guide.
+
+## 3DS compatibility
+
+This guide is compatible with every retail device in the Nintendo 3DS family of consoles (including the New 3DS series and the 2DS), regardless of region or firmware. 
+
+## Risks of installing custom firmware
+
+By modding your device, you subject it to the remote (but non-zero) possibility of the console being banned from online play or bricked (rendered permanently non-functional). **Incorrect file placement will not brick your device**, but purposely skipping instructions might. Similarly, the last ban wave occurred over five years, but there is no guarantee that another one will not occur in the future.
+
+After custom firmware is installed, it is possible for a system update to occur that custom firmware is unable to account for. Should you install the update, your 3DS may be rendered **temporarily** unable to boot until Luma3DS is updated. You can get up-to-date information about system updates on the homepage of this guide.
```

Comment:
maybe "homepage or [FAQ](faq) of this guide"

Reply:
now that I think about it, is this information strictly necessary? This is construed as a risk and something like this might be better suited for the FAQ or Finalizing Setup